src/main/java/com/github/maven_nar/Msvc.java

```diff
@@ -132,6 +132,12 @@ public void configureCCTask(final CCTask task) throws MojoExecutionException {
       envVariable.setKey("TMP");
       envVariable.setValue(getTempPath());
       task.addEnv(envVariable);
+      final String envInclude = System.getenv("INCLUDE");
```

Comment:
The doc does indeed say that `CL` will use `INCLUDE` and that it should be set to the VC include directory.

Experimentation so far is showing that `INCLUDE` doesn't really need to be set.

The change you have made though isn't setting the `INCLUDE` environment for the `CL` execution, rather it is taking the existing environment and adding it to the `CL` command line, thus overriding part of the point of discarding the environment to run the tools (more) cleanly.

I don't wish to just break whatever you got working, but I could use more of an idea of what includes where actually missing for you.